docker-redis/Makefile

```
@@ -1,3 +1,6 @@
+SHELL := /bin/bash
+.PHONY: build start connect run stop clean logs ps help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)